src/common/memory_tracking.hpp

```diff
@@ -1,5 +1,6 @@
 /*******************************************************************************
 * Copyright 2018-2024 Intel Corporation
+* Copyright 2024 Arm Ltd. and affiliates
```

Comment:
Seems to be unneeded.

Reply:
I believe it should have been added in this previous PR https://github.com/oneapi-src/oneDNN/pull/1966